### DISCUSSION TITLE
How to upgrade on Mac

### DISCUSSION THREAD

**Discussion by marksink** [autobrr/autobrr#1136] [Q&A] [answered]
Running straight from terminal:
 /autobrr/autobrr

I see there is a database, a config file, maybe other things that probably need to stay as is.

How should I go about upgrading to the latest version. There is nothing in ~/.config or ~/.autobrr

**Comment by KyleSanderson** [comment, marked as answer]
Simply drag and drop the new binary, then execute it.

**Comment by marksink** [comment]
Thanks.. I didn't realize config and db are only created after executing autobrr and not part of the download.  I at least stopped autobrr first, and duplicated autobrr just in case.  Seems to be fine and upgraded.  Thanks again.
